internal/project/js/build.go

```diff
@@ -71,6 +71,8 @@ func (build *Build) CompileDockerfile(output *dockerfile.Output) error {
 		Description(fmt.Sprintf("builds %s", build.Name())).
 		From("js").
 		Step(step.Arg(nodeBuildArgsVarName)).
+		Step(step.Copy("/", "/generated").From("generate-files")).
```

Comment:
I think frontend in general already has support for stuff generated by Go, so do we need that many copy steps?

should `make generate` just take care of all of that?

Reply:
frontend does not compile if the file that is going to be generated by `common.Generate` is there. And we have the chain of dependencies which trigger the frontend compilation when `make generate` is run. So I make sure we bring the file in before `npm run build` is triggered.